documentation/Functions.md

```diff
@@ -26,12 +26,24 @@ The following functions are already implemented:
 ***
 
 ### installDevonfwIde <a name="installDevonfwIde"></a>
+This function allows you to install devonfw IDE in your environment.
 #### parameter
-1. The tools you want to install within the devonfw ide: string array
-2. Optional: The version of the ide to install
+You need two parameter for this function.
+
+Example: installDevonfwIde(1st-parameter, 2nd-parameter)
+
+1st-parameter: The tools you want to install within the devonfw ide. You can pass every tool that can be installed with the devonfwIde. It is of type string array. In this parameter, the tools you can pass are: java, mvn, node, npm, gradle, ionic jasypt, jenkins, ng, sonar and yarn. You can pass any tool based on your requirement. 
```

Comment:
You use different spellings of devonfw-ide in the text. This should be consistent over the whole text. I would prefer `devonfw-ide`, as this is used in the official guide

Reply:
I have changed all wording to devonfw-ide.